src/test/java/org/solid/testharness/config/TargetServerTest.java

```diff
@@ -64,7 +58,7 @@ public void parseTargetServer() throws Exception {
 //                "    solid-test:disableDPoP true ;\n" +
                 "    solid-test:setupRootAcl true .");
```

Comment:
Is this something we can maintain in separate files? Seems to be quite a lot, so it might be better to have as a project resource?

Reply:
We can discuss that as a wider issue for a future task. I am oscillating between the 2 approaches. Embedded strings have the benefit of encapsulating the whole test in one file whereas resource files  make the test code look cleaner and the RDF easier to read but somewhere else. Don't forget that the tests in this code (and related RDF fragments) are the unit testing of the harness and will never be seen by people developing tests of a spec. Does that make sense?

Reply:
It does make sense, and certainly, lets consider it for future refactoring. 

I'm inclined to have RDF in a file of its own as a standard pattern, and thus have a generic toolchain to support it, but if that isn't in place, then lets get it done first, then get the toolchain and then get it done.